### PR TITLE
Branding update

### PR DESCRIPTION
**Pull request by christiemolloy**
## Motivation
https://issues.redhat.com/browse/IUXD-74?jql=labels%20%3D%20integration-ux-dev%20AND%20assignee%20in%20(christiemolloy)

## What
function added to `masthead.js` and `aboutModal.js` to return different logos depending on the cluster type.

## Why
Logos need to be changed depending on the cluster type.

## How
function added to `masthead.js` and `aboutModal.js` to return different logos depending on the  cluster type.

## Verification Steps
1. Go to `masthead.js`
2. Go to `getLogo`
3. Add a new line to make the cluster type: `poc` `osd` `pds` `localhost`
4. Check that the corresponding logos match the cluster type

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

## Progress

- [x] Finished task
- [ ] TODO

## Additional Notes

Screen Shot 2019-12-19 at 10 44 47 AM
Screen Shot 2019-12-19 at 10 45 19 AM
Screen Shot 2019-12-20 at 10 55 28 AM
Screen Shot 2019-12-20 at 10 53 53 AM

### CODE REVIEW

**Comment by tiffanynolan** [comment]
Looking good. Just wanted to verify any changes to the About Modal as well for those logos too. Please refer to the prototype: https://marvelapp.com/1jjffegg/screen/63909572

File: src/pages/devResources/devResources.js [hunk 1]
```diff
@@ -166,9 +166,9 @@ const mapStateToProps = state => ({
 
 const ConnectedDevResourcesPage = connect(
   mapStateToProps,
-  mapDispatchToProps
+  mapDispatchToProps,
```
Comment (mfrances17) [inline src/pages/devResources/devResources.js]: I would not include this file in the PR, not sure what it's adding the changes should be limited to just about and masthead. I'm guessing both of these changes would probably be flagged by lint.

**Comment by christiemolloy** [comment]
@dlabaj is there any way to determine which test is failing? Can't seem to tell from CircleCI

**Comment by mfrances17** [comment]
> @dlabaj is there any way to determine which test is failing? Can't seem to tell from CircleCI

@christiemolloy hey christie... i just approved a PR that should fix this issue. if you rebase and pull in that change the issue should be resolved.

**Comment by christiemolloy** [comment]
@mfrances17 I tried pulling in the latest but the tests still seem to failing

**Pull request merged**